### ISSUE TITLE
MacOS: Ked spustim autogram cez url (extension) mam dve ikonky

### ISSUE THREAD

**Issue by pomali** [slovensko-digital/autogram#252]
Na MacOS, Autogram spustim cez extension, v Docku aj v app switcher overlayi mam dve ikonky Autogramu
<img width="157" alt="image" src="https://github.com/slovensko-digital/autogram/assets/1194439/36d452ed-2a69-485f-a787-9caf026af1f0">

**Comment by pomali** [comment]
problem moze byt sposobeny tym, ze url handler spusta appku cez ten handler `Autogram.app/Contents/MacOS/Autogram` v `main.scpt` ktory premiena url event na cli parametre pre `Autogram.app/Contents/MacOS/AutogramApp`

(mal som pocit, ze sme to uz raz vyriesili, ale asi nie)

**Comment by celuchmarek** [comment]
@pomali toto sa ti ešte deje? Ja mám na Macu normálne iba jednu ikonku.

**Comment by pomali** [comment]
Áno, keď spúšťam cez schema handler

**Comment by kiblik** [comment]
Dnes som narazil na rovnaku vec, ked som siel podpisovat veci na Slovensko.sk
Ked som vylistoval procesy zistil som, ze su 2:
```
$ ps uax | grep -i autogram
kiblik           78661  15.4  0.2 33984584  36068   ??  S     3:56PM   0:00.96 /Applications/Autogram.app/Contents/MacOS/Autogram
kiblik           78664   1.2  0.5 39680604  75776   ??  S     3:56PM   0:00.77 /Applications/Autogram.app/Contents/MacOS/AutogramApp --url=autogram://listen?protocol=http
kiblik           78710   0.0  0.0 34129340    744 s000  R+    3:56PM   0:00.00 grep --color=auto -i autogram
```
Ked som ich skusil postupne zabit, ikonky postupne zmizli resp. prestali byt aktivne (zmizla bodka pod nimi). Je to teda isto previazane s `Autogram` + `AutogramApp`.
Note: bezim verziu 2.3.1

**Comment by celuchmarek** [comment]
@kiblik jo, toto je vlastne očakávané správanie, ako už spomenul Pom. Problém je, že nevieme iným spôsob spustiť appku priamo cez URI hadnler na Mac OS. Preto je tam Autogram skript, ktorý na základe URI handlera a jeho parametrov spustiť AutogramApp. Nepamätám si už presne, čo a aký problém to tam spôsobuje - buď to posiela tú URL takým spôsobom, že to v Autograme nevieme priamo pochrúmať alebo je problém v tom, že tým handlerom chceme spúšťať GUI appku 🤔
